datadog_checks_dev/datadog_checks/dev/tooling/commands/validate_commands/metadata.py

```diff
@@ -174,6 +174,11 @@
     'exacore',
     'build',
     'prediction',
+    'watt',
+    'kilowatt',
+    'megawatt',
+    'gigawatt',
+    'terawatt',
```

Comment:
I'm so curious about what the agent would be monitoring that needs to be measured in TW

Reply:
🤷‍♂️ I just added that here because I could. I can remove it (let's not talk about `exacore` and `exbibyte` units 🙈 )